checkov/common/output/cyclonedx.py

```diff
@@ -196,7 +195,7 @@ def create_library_component(self, resource: Record | ExtraResource, check_type:
         package_name = resource.vulnerability_details["package_name"]
         package_version = resource.vulnerability_details["package_version"]
 
-        if purl_type == PURL_TYPE_MAVEN:
+        if purl_type == PURL_TYPE_MAVEN and '_' in package_name:
```

Comment:
@NoaAzoulay Can there be maven packages without `_`? Is it because we are using `:`?

Reply:
for some reason twistcli didn't find the group name for a specific Maven package, even it is has one in Maven. It was this one `bcpkix-jdk15on` which should have been `org.bouncycastle_bcpkix-jdk15on`

Reply:
@NoaAzoulay would be great, if you can add or extend a test to validate the new change 🙂

Reply:
@gruebel sure, working on it